src/lightning_app/cli/lightning_cli.py

```diff
@@ -116,6 +118,83 @@ def run_app(
     _run_app(file, cloud, without_server, no_cache, name, blocking, open_ui, env)
 
 
+@main.group()
+def exec():
+    """exec your application."""
+
+
+def _retrieve_application_url(app_id_or_name: Optional[str]):
+    failed_locally = False
+
+    if app_id_or_name is None:
+        try:
+            url = "http://127.0.0.1:7501"
+            resp = requests.get(f"{url}/api/v1/commands")
+            if resp.status_code != 200:
+                raise Exception(f"The server didn't process the request properly. Found {resp.json()}")
+            return url, resp.json()
+        except ConnectionError:
+            failed_locally = True
+
+    if app_id_or_name or failed_locally:
+        from lightning_app.utilities.cloud import _get_project
+        from lightning_app.utilities.network import LightningClient
+
+        client = LightningClient()
+        project = _get_project(client)
+        list_lightningapps = client.lightningapp_instance_service_list_lightningapp_instances(project.project_id)
+
+        lightningapp_names = [lightningapp.name for lightningapp in list_lightningapps.lightningapps]
+
+        if not app_id_or_name:
+            raise Exception(f"Provide an application name or id with --app_id_or_name=X. Found {lightningapp_names}")
+
+        for lightningapp in list_lightningapps.lightningapps:
+            if lightningapp.id == app_id_or_name or lightningapp.name == app_id_or_name:
+                resp = requests.get(lightningapp.status.url + "/api/v1/commands")
+                if resp.status_code != 200:
+                    raise Exception(f"The server didn't process the request properly. Found {resp.json()}")
+                return lightningapp.status.url, resp.json()
+    return None, None
+
+
+@exec.command("app")
+@click.argument("command", type=str, default="")
+@click.option("--args", type=str, default=[], multiple=True, help="Env variables to be set for the app.")
```

Comment:
`Env variables to be set for the app.`

maybe copied by mistake? 😄

Reply:
Done !